prog/setup_grafana.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class Prog::SetupGrafana < Prog::Base
+  subject_is :sshable
+
+  def self.assemble(sshable_id, grafana_domain:, certificate_owner_email:)
+    if grafana_domain.strip.empty? || certificate_owner_email.strip.empty?
```

Comment:
It's better to avoid duplication (currently, `strip` is called twice for each variable):
```suggestion
    grafana_domain = grafana_domain.strip
    certificate_owner_email = certificate_owner_email.strip
    if grafana_domain.empty? || certificate_owner_email.empty?
```